ezmodel/ezmodel.py

```diff
@@ -36,14 +36,16 @@ def regularization_plot():
                                 LogisticRegression(), Ridge(), Lasso().
                                 Can also be a pipeline with several steps containing one of the above models.
 
-        alpha (): Penalty constant multiplying the regularization term. Larger value corresponds to stronger
+        alpha (float or list): Penalty constant multiplying the regularization term. Larger value corresponds to stronger
                   regularization. Can be list or float.
 
```

Comment:
Maybe just be a bit more specific about what the returns are for float and list inputs to `alpha`